extensions/Cake.MiniCover.yml

```diff
@@ -9,4 +9,5 @@ Description: Cake addin for MiniCover
 Categories:
 - code coverage
 - testing
-TargetCakeVersion: 0.29.0
+TargetCakeVersion: 0.28.0
```

Comment:
The change can be explained by the fact that we are now analyzing the most recent stable release:

![image](https://user-images.githubusercontent.com/112710/101060151-1118ca00-355d-11eb-8e42-6ab8d40f6658.png)